immunopepper/mode_cancerspecif.py

```diff
@@ -92,6 +90,7 @@ def mode_cancerspecif(arg):
                                                                           or (arg.filterNeojuncCoord == 'A') else False,
                                               filterAnnotatedRF=True if (arg.filterNeojuncCoord == 'N')
                                                                          or (arg.filterNeojuncCoord == 'A') else False,
+                                              separate_back_annot=True,
```

Comment:
Maybe you can add a comment what `separate_back_annot` means - or find a more descriptive name.